themes/contribute.jquery.org/cla-check.php

```diff
@@ -110,7 +110,14 @@ function getProcessedPost( $data ) {
 function neglectedAuthors( $data ) {
 	$html = "<ul>\n";
 	foreach ( $data->data->neglectedAuthors as $author ) {
-		$html .= "<li>" . htmlspecialchars( "$author->name <$author->email>" ) . "</li>\n";
+		$html .= "<li>" . htmlspecialchars( "$author->name <$author->email>" );
+		if ( count( $author->errors ) ) {
+			$html .= ": ";
+			foreach( $author->errors as $error ) {
```

Comment:
Don't know if we really want to care about PHP code style (I don't think we ever really did) but on line 112 there is a space after the `foreach`.